utilities/backupable/backupable_db_test.cc

```diff
@@ -1994,12 +1941,18 @@ TEST_F(BackupableDBTest, ShareTableFilesWithChecksumsNewNaming) {
     CloseDBAndBackupEngine();
     AssertBackupConsistency(1, 0, keys_iteration, keys_iteration * 2);
     AssertDirectoryFilesMatchRegex(backupdir_ + "/shared_checksum",
-                                   std::regex(pair.second),
+                                   std::regex(pair.second), ".sst",
                                    1 /* minimum_count */);
     if (std::string::npos != pair.second.find("_[0-9]+[.]sst")) {
       AssertDirectoryFilesSizeIndicators(backupdir_ + "/shared_checksum",
                                          1 /* minimum_count */);
     }
+
+    if (options_.enable_blob_files) {
```

Comment:
We could call `AssertDirectoryFilesMatchRegex` unconditionally since `enable_blob_files` is always set (same in `ShareTableFilesWithChecksumsOldFileNaming` below)